pkg/ovs/ovn-nb-logical_router_policy.go

```diff
@@ -0,0 +1,77 @@
+package ovs
+
+import (
+	"fmt"
+	ovsclient "github.com/kubeovn/kube-ovn/pkg/ovsdb/client"
+	"github.com/kubeovn/kube-ovn/pkg/ovsdb/ovnnb"
+	"github.com/ovn-org/libovsdb/model"
+	"github.com/ovn-org/libovsdb/ovsdb"
+)
+
+func (c OvnClient) AddRouterPolicy(lr *ovnnb.LogicalRouter, matchfield string, action ovnnb.LogicalRouterPolicyAction,
+	opts map[string]string, extIDs map[string]string, priority int) error {
+	lrPolicy := &ovnnb.LogicalRouterPolicy{
+		Action:      action,
+		Match:       matchfield,
+		Options:     opts,
```

Comment:
should set external_ids not options

Reply:
external_id is listed here for a complete installation. Actually it is not used.